aws/sagemaker/large-model-inference/sample-llm/text_embedding_deploy_bert.ipynb

```diff
@@ -94,18 +86,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# model_id = \"djl://ai.djl.huggingface.onnxruntime/BAAI/bge-base-en-v1.5\"\n",
-    "model_id = \"BAAI/bge-base-en-v1.5\"\n",
+    "# model_id = \"djl://ai.djl.huggingface.onnxruntime/BAAI/bge-base-en-v1.5\" # use DJL model zoo model\n",
+    "# model_id = \"s3://YOUR_BUCKET\" # download model from your s3 bucket\n",
+    "model_id = \"BAAI/bge-base-en-v1.5\" # model will be download form Huggingface hub\n",
     "\n",
     "env = {\n",
-    "    \"HF_MODEL_ID\": model_id,\n",
-    "    \"OPTION_ENGINE\": \"OnnxRuntime\",\n",
     "    # \"SERVING_BATCH_SIZE\": \"32\", # enable dynamic batch with max batch size 32\n",
-    "    \"SERVING_MIN_WORKERS\": \"1\", # make sure min and max Workers are equals when deploy model on GPU\n",
+    "    \"SERVING_MIN_WORKERS\": \"1\",   # make sure min and max Workers are equals when deploy model on GPU\n",
     "    \"SERVING_MAX_WORKERS\": \"1\",\n",
+    "    # \"OPTION_OPTIMIZATION=O2\",   # use OnnxRuntime O2 optimization\n",
     "}\n",
     "\n",
-    "model = Model(image_uri=image_uri, env=env, role=role)"
+    "model = DJLModel(\n",
+    "    model_id=model_id,\n",
+    "    engine=\"OnnxRuntime\",\n",
```

Comment:
should we use `task=text-embedding` here instead of `engine` in case we update/auto set different engines for embedding use-case?

Reply:
good catch, fixed